action.yml

```diff
@@ -79,6 +79,10 @@ runs:
 
         set -e
         
+        if [ -z "${source_image}" ]; then
```

Comment:
lets add a message as well, mentioning that resolving all mirrors failed.

Reply:
Actually this won't work if there are no registry mirrors configured on the platform, cause in that case `source_image` will be an empty string.

Reply:
Discussed with @alexeysas - and since we are always creating the `resolved-image.file` file (regardless of mirrors being defined or not), `source_image` will indeed be empty if any of the crane commands in the file processing loop error out.